envutil/envutil.go

```diff
@@ -41,3 +44,34 @@ func GetenvWithDefault(envKey, defValue string) string {
 	}
 	return retVal
 }
+
+// RequiredEnv - returns the env's value if specified, otherwise it returns an error that explains the key is required.
```

Comment:
The format of the comment section is not godoc compatible, I think there are missing indentations. Read more here: https://www.ardanlabs.com/blog/2013/06/documenting-go-code-with-godoc.html

Also the problem visually:
![screen shot 2019-03-07 at 10 25 17 am](https://user-images.githubusercontent.com/1564933/53946179-8a069b80-40c3-11e9-9f94-83e57079f9c0.png)